python/src/trezorlib/cli/benfen.py

```
@@ -0,0 +1,60 @@
+
+from .. import benfen, tools
+from . import with_client
+
+if TYPE_CHECKING:
+    from ..client import TrezorClient
+
+PATH_HELP = "BIP-32 path, e.g. m/44'/728'/0'/0'/0'"
+
+
+@click.group(name="benfen")
+def cli():
```

Ruff (0.8.2)

Missing return type annotation for public function `cli`

Add return type annotation: `None`

(ANN201)